DB/models/movies.js

```diff
@@ -1,18 +1,20 @@
 import { Schema, model, models } from "mongoose";
 
 const moviesSchema = new Schema({
-    id: Number,
-    title: String,
-    date: String,
-    img: String,
-    details: {
-        about: String,
-        director: String,
-        writers: String,
-        stars: String,
-        rating: Number
-    }
+  id: Number,
+  title: String,
+  date: String,
+  img: String,
+  details: {
+    about: String,
+    director: String,
+    writers: String,
+    stars: String,
+    rating: Number,
+  },
```

Comment:
It's always a good idea to agree amongst the team on formatting (and enforce it through automation, e.g. using [Prettier](https://prettier.io/)) so that pull requests will not contain changes to files that don't actually change anything besides style/formatting.